platforms/hanwha-wn/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+include(ExternalProject)
+
+set(BOOST_BUILD_PATH ${CMAKE_CURRENT_BINARY_DIR}/boost-prefix/src/boost)
+
+file(WRITE ${BOOST_BUILD_PATH}/user-config.jam "using gcc : arm : \"$ENV{WN}/arm-linux-gnueabi-g++\"")
+
+ExternalProject_Add(boost
+    URL https://sourceforge.net/projects/boost/files/boost/1.63.0/boost_1_63_0.tar.bz2
+    URL_HASH SHA1=9f1dd4fa364a3e3156a77dc17aa562ef06404ff6
+    BUILD_IN_SOURCE 1
+    UPDATE_COMMAND ""
+    PATCH_COMMAND ""
+    CONFIGURE_COMMAND ./bootstrap.sh
+    BUILD_COMMAND ./b2 install
```

Comment:
I looked what people do with it and there is no solution for GDrive it seems. People say it can work with DropBox. We have DropBox, so lets try it.